sidebar_translations.json

```
@@ -56,7 +56,7 @@
   "develop.data_generation.block_models": "Block Model Generation",
   "develop.rendering": "Rendering",
   "develop.rendering.basic_concepts": "Basic Rendering Concepts",
-  "develop.rendering.draw_context": "Using the Drawing Context",
+  "develop.rendering.draw_context": "Using GuiGraphics",
```

I think we try to avoid class names in page titles because they are meaningless for newcomers.

also, ditto with the key:

```suggestion
  "develop.rendering.gui_graphics": "Using GuiGraphics",
```